repos/system_upgrade/el7toel8/actors/biosdevname/actor.py

```diff
@@ -9,7 +9,7 @@
 
 class Biosdevname(Actor):
     """
-    Enable biosdevname on RHEL8 if all interfaces on RHEL-7 used biosdevname naming scheme or if machine vendor is DELL
+    Enable biosdevname on RHEL8 if all interfaces on RHEL-7 used biosdevname naming scheme and if machine vendor is DELL
```

Comment:
> [C0301 line-too-long] Line too long (120/119) File: repos/system_upgrade/el7toel8/actors/biosdevname/actor.py, line 12

;-(

Reply:
Using present tense in both parts of the statement fixes the 'problem' :)
```suggestion
    Enable biosdevname on RHEL8 if all interfaces on RHEL-7 use biosdevname naming scheme and if machine vendor is DELL
```

Reply:
@vojtechsokol unfortunately that's not correct because it would have to be: 

if all interfaces on RHEL-7 use **the** biosdevname naming scheme